src/main/java/edu/kis/powp/jobs2d/drivers/command/ComplexCommand.java

```diff
@@ -0,0 +1,24 @@
+package edu.kis.powp.jobs2d.drivers.command;
+
+import java.util.ArrayList;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class ComplexCommand implements DriverCommand {
+
+    ArrayList<DriverCommand> commandsArray = new ArrayList<DriverCommand>();
```

Comment:
Should be private. The type of variable should point to interface (i.e., List<>), not an implementation.